assets/css/style.css

```diff
@@ -0,0 +1,123 @@
+@import url('https://fonts.googleapis.com/css?family=Cabin&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Bangers&display=swap');
+
+body {
+    color: whitesmoke !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/app/columbia-fsf-f2019/trivia-game/pullRequest?prid=4555494)